Dockerfile

```diff
@@ -3,8 +3,16 @@ FROM python:3.7-alpine3.10
 MAINTAINER Paul Podgorsek <ppodgorsek@users.noreply.github.com>
 LABEL description Robot Framework in Docker.
 
+# Set the reports directory env variable
+# By default, the directory is /opt/robotframework/reports
+ENV ROBOT_OUTPUT_DIR /opt/robotframework/reports
+
+# Set the tests directory env variable
+# By default, the directory is /opt/robotframework/tests
+ENV ROBOT_TESTS_DIR /opt/robotframework/tests
+
 # Setup volume for output
-VOLUME /opt/robotframework/reports
+VOLUME ${ROBOT_OUTPUT_DIR}
```

Comment:
do we also need a volume for ROBOT_TESTS_DIR?

Reply:
we have not included a volume for tests as certain use cases involve people installing the tests via another docker image that is based on this and so the tests are not in a volume. -v will of course work just fine either way